cpp/tensorrt_llm/cortex.tensorrtllm/examples/server/dylib.h

```diff
@@ -0,0 +1,316 @@
+/**
```

Comment:
I copied `dylib.h` and `httplib.h` files from `cortex.llamacpp` in order to prepare for example server.